AvatarChatbot/docker/gaudi/README.md

```diff
@@ -0,0 +1,226 @@
+# AvatarChatbot Application
+
+The AvatarChatbot service can be effortlessly deployed on either Intel Gaudi2 or Intel XEON Scalable Processors.
+
+## 🚀 Megaservice flow
+
+```mermaid
+flowchart LR
+    subgraph AvatarChatbot
+        direction LR
+        A[User] --> |Input query| B[AvatarChatbot Gateway]
+        B --> |Invoke| Megaservice
```

Comment:
```suggestion
        B <-->  Megaservice
```

Reply:
Replaced with new mermaid flowchart for AvatarChatbot